content/blog/2023-01-26-ethereum-classic-course-6-ethereum-classic-is-the-original-chain/index.md

```diff
@@ -0,0 +1,132 @@
+
+Consequently, as we will see in this class, Ethereum Classic is the philosophy of Bitcoin with the technology of Ethereum.
+
+## Illustration of the Ethereum Split From Ethereum Classic 
+
+![A step by step explanation of the ETH fork.](./etc-course-eth-split.png)
+
+Ethereum Classic and Ethereum were one blockchain when it was founded in 2015. Then, Ethereum split from Ethereum Classic on July 20th of 2016, on block 1,920,000, as they implemented the reversal of funds through an irregular state change.
+
+Today, ETC is the original chain with an uninterrupted history of transactions, running the proof of work consensus mechanism, with a fixed monetary policy, and a Code Is Law philosophy. ETH has a Social Consensus philosophy, no monetary policy, its history has been changed and may be changed at any time, and runs the proof of stake consensus mechanism.
+
+## Ethereum Classic’s Design
+
+![ETC](./etc-course-etc-logo.png)
+
+As said above, Ethereum Classic uses the same base technology as Bitcoin but added programmability.
+
+The Ethereum Classic ledger stores accounts and balances as Bitcoin, but adds software programs which become decentralized, also called smart contracts, when they are sent to the network.
+
+To accomplish this it has 6 major components:
+
+1. The Ethereum Virtual Machine (EVM)
+2. A programming language
+3. State transition
+4. The gas system
+5. The ETChash mining algorithm
+6. A fixed monetary policy
+
+## 1. The Ethereum Virtual Machine (EVM)
+
+The EVM is a software component that behaves like a computing machine. As part of the node software, it is replicated in all machines in the network, therefore it’s a decentralized virtual machine.
+
+The way it works is that it has over 120 computing operation codes, or opcodes for short, that together provide near full computability.
+
+Because node software clients may be installed in many kinds of computers with diverse operating systems, the EVM enables nodes to operate in their local environments with their local machines, and at the same time interact with the global network of blockchain nodes with a single and compatible computing standard.
+
+This assures a unified way of operating the network and executing smart contracts inside the secure environment of Ethereum Classic.
+
+## 2. A Programming Language
+
+For software programs to be sent and stored in the network, and executed by the EVM, there had to be some programming languages that would be compatible and adapted for a blockchain environment.
+
+Solidity was one of the programming languages that were created to write programs for Ethereum, and remains to this day the most popular one for building dapps on Ethereum and Ethereum Classic.
+
+It is syntactically similar to JavaScript, which is widely used by many developers around the world, but it is adapted so it may work on a decentralized peer-to-peer setting.
```

Comment:
Line 82: Usage
```suggestion
It is syntactically similar to JavaScript which is widely used by many developers around the world but has been adapted so it may work in a decentralized peer-to-peer setting.
```